### DISCUSSION TITLE
MapLayersUiItemsProvider component not support TileURL type when Adding new layers

### DISCUSSION THREAD

**Discussion by ghost** [iTwin/itwinjs-core#6503] [General] [answered]
Why does the MapLayersUiItemsProvider component not support TileURL type when creating new layers in the list?
```
import { MapLayersUI, MapLayersUiItemsProvider } from '@itwin/map-layers';
const uiProviders = [
  new ViewerNavigationToolsProvider(),
  new MapLayersUiItemsProvider({
    hideExternalMapLayers: false,
    mapTypeOptions: { supportTileUrl: true, supportWmsAuthentication: true },
    fetchPublicMapLayerSources: false
  })
];
```
![addNewLayer](https://github.com/iTwin/itwinjs-core/assets/34014227/680a329c-5aa7-4a73-b889-ca1abdda6d57)

**Comment by pmconne** [comment]
@mdastous-bentley

**Reply by mdastous-bentley** [reply, marked as answer]
TileURL option is hidden by default in the MapLayers widget.  It can be enabled when creating the `MapLayersUiItemsProvider`.  I just fixed an issue about it, will be available in the next version.

**Reply by ghost** [reply]
thank you very much.
